experimental/uefi/kernel/src/boot.rs

```diff
@@ -0,0 +1,48 @@
+//
+// Copyright 2022 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use strum::{Display, FromRepr};
+
+#[derive(Debug, PartialEq, Eq, Display, FromRepr)]
+#[repr(u32)]
+/// E820 address range types according to Chapter 15 of th ACPI Specification, Version 6.3.
```

Comment:
Perhaps also add a link to it? e.g. https://uefi.org/specs/ACPI/6.4/15_System_Address_Map_Interfaces/Sys_Address_Map_Interfaces.html (BTW this is v 6.4,)

Reply:
Done. I missed the fact there's a newer version available.